src/ParseServer.js

```diff
@@ -467,10 +467,6 @@ function injectDefaults(options: ParseServerOptions) {
       });
     }
   });
-
-  options.masterKeyIps = Array.from(
```

Comment:
Can this really just be removed and the issue is fixed?

cc @dblythy

Reply:
I believe so. I will test and report back